### ISSUE TITLE
Option to show share button in Jupyter widget

### ISSUE THREAD

**Issue by wybert** [keplergl/kepler.gl#2284]
There is no share button in Kepler.gl Jupyter widget,

<img width="446" alt="image" src="https://github.com/keplergl/kepler.gl/assets/5254627/a840f0c5-540d-496f-a311-d98b8ae96e67">

But there are share button in kKepler.gl website, https://kepler.gl/demo/world_flights

<img width="285" alt="image" src="https://github.com/keplergl/kepler.gl/assets/5254627/72134512-3a02-437e-8ae3-a784077bef96">

Is it supported in Jupyter, how can I do it

**Comment by 861** [comment]
Could you also add the export option to the Jupyter widget?

**Comment by wybert** [comment]
Any updates for this?

**Comment by Bo-Brown-Dev** [comment]
Would love to see this tackled, It'd be very helpful

**Comment by cartesian-theatrics** [comment]
I agree. It looks like the last update to keplergl was in Sep 2021, is it not supported/maintained anymore?
